pytest_asyncio/plugin.py

```diff
@@ -139,7 +140,8 @@ def pytest_pyfunc_call(pyfuncitem):
     function call.
     """
     for marker_name, fixture_name in _markers_2_fixtures.items():
-        if marker_name in pyfuncitem.keywords:
+        if inspect.isasyncgenfunction(pyfuncitem.obj) \
```

Comment:
`pytest-asyncio` claims that the minimum supported Python is 3.5.
`isasyncgenfunction` is available in Python 3.6+ only

Reply:
Fixed :smile: